service/controller/v1/prometheus/prometheus.go

```diff
@@ -37,6 +37,22 @@ var (
 	// KubernetesSDServiceNameLabel is the label applied to the target
 	// by Prometheus Kubernetes service discovery that holds the target's Kubernetes service.
 	KubernetesSDServiceNameLabel = model.LabelName("__meta_kubernetes_service_name")
+
+	// KubernetesSDServiceGiantSwarmMonitoringPresentLabel is the label applied to the target
+	// by Prometheus Kubernetes service discovery that holds the target's Kubernetes service presenting the label giantswarm_io_monitoring.
+	KubernetesSDServiceGiantSwarmMonitoringPresentLabel = model.LabelName("__meta_kubernetes_service_labelpresent_giantswarm_io_monitoring")
```

Comment:
Dumb question. Why `giantswarm_io_` and not just `giantswarm_`?

Reply:
`__meta_kubernetes_service_lablelpresent_<label_name>` checks whether label is on the service object, and we would check `giantswarm.io/monitoring` in this example. (`/` and `.` are replaced with `_` since they are not allowed in prom label formats)

Reply:
Ack. Thanks for the explanation.